DEPS

```diff
@@ -6,7 +6,7 @@ vars = {
   'effcee_revision': 'b83b58d177b797edd1f94c5f10837f2cc2863f0a',
   'googletest_revision': '2f42d769ad1b08742f7ccb5ad4dd357fc5ff248c',
   're2_revision': 'e356bd3f80e0c15c1050323bb5a2d0f8ea4845f4',
-  'spirv_headers_revision': '123dc278f204f8e833e1a88d31c46d0edf81d4b2',
+  'spirv_headers_revision': 'e4322e3be589e1ddd44afb20ea842a977c1319b8',
```

Comment:
This is now obsolete 
See https://github.com/KhronosGroup/SPIRV-Tools/pull/2880